workspace/exercise_metadatas.go

```diff
@@ -0,0 +1,18 @@
+package workspace
+
+// ExerciseMetadatas is a collection of exercise metadata to interactively choose from.
+type ExerciseMetadatas []*ExerciseMetadata
```

Comment:
I'm not super happy about `Metadatas` as a plural. Data is already plural. How about something like `MetadataCollection` or `ExerciseMetadataCollection`?